spmd/tensor/parallel/api.py

```diff
@@ -89,6 +94,92 @@ def _shard_self_attn_params(name: str, module: nn.Module) -> None:
                 module.register_module(n, tp_multi_head_attention)
 
 
+# Define partition functions needed to parallelize Linear modules
+def _rowwise_parallelize_fn(name, module, device_mesh):  # pyre-ignore[2, 3]
```

Comment:
let's probably add linear here. Because for other types of module, rowwise and colwise will be different.

Reply:
To be clear, previous this is inner function inside `_parallize_mlp` so that we don't need to add linear in the name. Also please do add in/o type and comments for this function. Thanks!

Reply:
Thx for catching it!